codex-process-feasibility/src/test/java/de/netzwerk_universitaetsmedizin/codex/processes/feasibility/EnhancedFhirWebserviceClientProviderImplTest.java

```diff
@@ -0,0 +1,76 @@
+package de.netzwerk_universitaetsmedizin.codex.processes.feasibility;
+
+import org.highmed.dsf.fhir.client.FhirWebserviceClientProvider;
+import org.highmed.fhir.client.FhirWebserviceClient;
+import org.hl7.fhir.r4.model.IdType;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertSame;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
+public class EnhancedFhirWebserviceClientProviderImplTest {
+
+    @Mock
+    private FhirWebserviceClient client;
+
+    @Mock
+    private FhirWebserviceClientProvider clientProvider;
+
+    @InjectMocks
+    private EnhancedFhirWebserviceClientProviderImpl enhancedFhirWebserviceClientProvider;
+
+    @Test
+    public void testGetWebserviceClient_Local() {
```

Comment:
There is a second local case when the baseURl is the same as the providers baseURL. Please add another test.